src/rgw/rgw_sal_dbstore.cc

```diff
@@ -722,6 +720,11 @@ namespace rgw::sal {
     return op_target.obj_omap_set_val_by_key(dpp, key, val, must_exist);
   }
 
+  int DBObject::chown(User& new_user, const DoutPrefixProvider* dpp, optional_yield y)
+  {
+    return 0;
```

Comment:
I felt that Objcet::chown didn't need to be implemented by a Store, so it's okay to return success here.  I don't have a strong opinion, except that this will make rgwadmin chown fail.

Reply:
i think it's much better to fail with an obvious "Operation not supported" error than to succeed without actually changing ownership - that would just confuse users